docs/Tooling.md

```diff
@@ -27,6 +27,9 @@ For Flycheck:
 
 To start an interactive session, make sure `carp` is in your path (inside Emacs) and execute `M-x run-carp`.
 
+# Atom
+[language-carp](https://atom.io/packages/language-carp) offers both TextMate and Tree-sitter, where tree-siiter grammar is more powerful
```

Comment:
There's a typo there (siiter)

Reply:
There is no typo in 'sitter', pro project call tree-sitter, look out their site https://tree-sitter.github.io/tree-sitter/

Reply:
There is, continue reading after the comma.

Reply:
Yeah, sorry, misted that, I was without glasses xD
Was hard to spot on